tgui/packages/tgui/interfaces/JobSelection.tsx

```diff
@@ -0,0 +1,185 @@
+import { useBackend } from '../backend';
+import { Box, Button, StyleableSection, Icon, Stack } from '../components';
+import { Window } from '../layouts';
+import { Color } from 'common/color';
+import { SFC } from 'inferno';
+import { JobToIcon } from './common/JobToIcon';
+import { deepMerge } from 'common/collections';
+
+type Job = {
+  unavailable_reason: string | null;
+  command: boolean;
+  open_slots: number;
+  used_slots: number;
+  icon: string;
+  prioritized: boolean;
+  description: string;
+};
+
+type Department = {
+  color: string;
+  jobs: Record<string, Job>;
+  open_slots: number;
+};
+
+type Data = {
+  departments_static: Record<string, Department>;
+  departments: Record<string, Department>;
+  alert_state: string;
+  shuttle_status: string;
+  disable_jobs_for_non_observers: boolean;
```

Comment:
`BooleanLike`?

Bools do not exist in BYOND and it just sends them as numbers, so we use BoolenaLike to paper over it.